src/main/java/org/terasology/metalrenegades/world/SimplexHillsAndMountainsProvider.java

```diff
@@ -49,21 +49,16 @@ public void process(GeneratingRegion region) {
 
         float[] mountainData = mountainNoise.noise(facet.getWorldArea());
         float[] hillData = hillNoise.noise(facet.getWorldArea());
-        SurfaceTemperatureFacet temperatureData = region.getRegionFacet(SurfaceTemperatureFacet.class);
-        SurfaceHumidityFacet humidityData = region.getRegionFacet(SurfaceHumidityFacet.class);
+        float[] mIntensData = mountainIntensityNoise.noise(facet.getWorldArea());
```

Comment:
I would rather use the full name `mountainIntensityData` here instead of an abbreviation for better readability